sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -553,6 +535,8 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
         ReloadWorkflowTasks.webpackTask(fastOptJS)
       }.value,
 
+      installWebpackDevServer := npmUpdate.value,
```

Comment:
Yes and also no to break user's code. Let's keep it until we have a breaking release?